assets/scripts/cookie-consent.js

```diff
@@ -3,21 +3,34 @@
     return;
   }
 
+  function addCookieSettingsButton() {
```

Comment:
The function name is misleading as this function doesn't actually add a button but registers the click handler for it. IMO this doesn't necessarily needs to be in a dedicated function as it's invoked once. But we can leave it to move forward 👍

Reply:
referenced in issue and created a ticket for it to fast forward. #51